docs/notebooks/tutorial.ipynb

```diff
@@ -575,356 +547,6645 @@
zsqKu16kk0iuueL/HL8A57p/EyafQ884JZh6TK5NPE/dAi6dYPOnd3Dk+KpYCXdcdba\n5LmWjDERZBy3W0RERCTk7d4NbdsW3PX27Enbo+0XGZnyPiExge7TuvPOb++wZOgSTi95eppjs23P\n/v3Qvr3rPn/2WQ0PGAKClXQvNsY8CJQ0xnQCZgCzg3QvCWGq7fOOYu8txd9bir93TsXYf/edG/Aj\nNtZNZpOfQdF++QV++CFlGMDU/El3kk1iyCdDCDfhLLtpGeecfk6GY7t2dSMAprdo0SJ49103huDl\nl8Mbb0CExs0IBcH6KdwPDANWAbcAnwP/C9K9RERERILi0CFXEg3wxx9w//3Qty/Urp236/knwSlR\nIuO+qCiITYhl+NzhbDu4jXnXzSMiLHCqlqHzevt2GDPGLWfMgNat89ZICYqgjNNtjCkFHLfWJvrW\nw4Fi1tpjBX6z7NuicbpFREQkT1atchUaAJ995kqkL7sM5s3LWE+dlJRxW3qnneamfT90yPV2R0bC\niRPQpAk8/foOxq4dSkRYBNN6T6N88fI5a+SBA244wDffhBtvhNGj3XSWctJlNU53sMpLvgZS/w1X\nEpgfpHuJiIiIBMUff0CVKnD66S7hBliwwFVwjB8PiYlu2wsvQLly2V+vdm246CI3kyTAQw/Bv/8N\nk+au5ubv2tOyWks+6vtRzhPuL75wwwDGxLjGjh+vhDtEBSvpLmatPeJfsdYexiXeUsScirV9hYVi\n7y3F31uKv3dOhdjv2AEbNrj3a9bADTdAhQppj/nsM7jvPvjwQ7f+1ltw5Igr+bj1Vrft+HGIi0s5\nx1o3ZfusWSmlIcP/s58Nza+i85TO3H3R3YzpMIaSkTlImbZuhWHDXM/222/D5MlQpcopEf9TVbCS\n7qPGmKb+FWNMMyA2SPcSERERKTBdurip18FN5Fi/PtSo4db//W+33LPHLX/91S1TV7K+9ppbb9gw\n7eSPGza4kpIqVdz6R2s+ot7L9ahVrhZb7trC7S1uz1kDJ0509SgVK8LKldCxY94+qJxUwarpbg68\nD/zl21QVuNZa+3OB3yz7tqimW0RERHLMX3d9yy3w+uvwzTfw0kvu2cQTJ6BHD5dA793r8t3p0924\n2xMmwKHDiQx4YgbXP/IVk+eugbI76dHiQlpEt+DY+las+aoFM6YV58EFD/L+7+8z/ZrptKzeMueN\nGzvWJd0ffZRSbC4h46TXdFtrfwLqA7cC/wLO9SLhFhEREcmN48ddwg0u4QZo2RL++ce9Dw+Hs86C\n9euhenWYP98l4gcPwqEqnzFq73mEX/QKx7dcSLU1T8M7C7mmXn/+Of4P7+56mDn1K1N8THGW717O\n8luW5y7hHj/eJdxffqmEuxAq0KTbGHOZb9kb6AbUBeoB3Y0xvQryXlI4qLbMO4q9txR/byn+3ins\nsV+3zpWWpJ68JioK6tVLuw5u2L/TT4fnPvyWg1d14sEldzK+83hOvLGE6ffewZ6f21Cz7Fk0SOrH\n+M7jGXj8Wx6OPMDxh44z7/p5VCiRrlA8Ky+95F4LFkCdOgEPK+zxP5UVdE93O9+yu+/Vzffyr4uI\niIiErNWrXSfyiRNuvW9ft3z2Wdi3z72Pj3fLp95ZxdGeV/CfHwfAml78cccfdKvbLXmSm4QEd60d\nO9z6a69BRFhklmNvZ+qNN+DJJ2HxYjjzzPx9QPFMgdd0G2PCgD7W2g8K9MJ5pJpuERERyam6dd2Q\nfjNmuFKThISMEzqasCRo/C5nDLyHriUfZspdN8OJ4skPU15yiasDB1f/vXcvzJwJVau6oQYHDsxh\nY5KS3BSY/h7u+vUL7HNKcGRV0x2sByl/sdY2zf7I4FPSLSIiIjl1wQXwv/9B8+ZuPX0KEXMkhip3\nXQ3hCfw88nUqJzVNHtnEf+zx426697g4l3w/+mjK+TlOSWJj4frrYds2+PhjiI7O1+eSk8OLyXG+\nMsbca4ypYYyp4H8F6V4SwlRb5h3F3luKv7cUf+8U5tgnJcGff6bUb5cqlXb//tj9tJ7YmuHdO7Dq\n/5bRNLop1au7IQD9iTe4YQJbt4YOHaBt2zw0JD7eJdyJiS5rz0XCXZjjf6rLZVFRjvUDLJB6wEkL\nBK78FxEREfHIs8/CPfe492XKuGqOEqnm1k5MSuS6mdfRsU5HXuz2eJpzf/2V5Dru9M46K5cNOXgQ\nrrvO9XTPnp12oG8p1IJSXhJKVF4iIiIigcTGwtNPw8iRKdvSpw1JNolrpl/DvmP7+Or6rygWUSxX\n93j3Xddx3bmzG+0voB9/hMGD3RiFr72WNuuXQiGr8pKg9HQbY0oAtwFtcD3cS4AJ1trjwbifiIiI\nSFb27IFKldwMk1u3Qu/ebnvJVDOuHzuW+bnP//A8Ww9u5dsbvs11wg0pww+OHx/gAP8Dk889By+8\nANdeC2HBqgAWrwTrJzoZaAC8CLwMnAdMCdK9JISptsw7ir23FH9vKf7eCcXY797tEt+EBPeA5DXX\nwF9/wZ13uv0jRrje7RIlMnYubzu4jTHfjOGDaz6geETeSj0uvNAtM60UOXIErv7/9u47PKpqa+Dw\nb6cnEHroHQRp0kEQISBNQEAQUFGaYMGCXlFRr168lk+siFzFitJEQBSkSwkgVRAQpPfeW0L6ZH9/\nrMRJIIFUTsp6n2eemTlz5pwziyGzZs86a3eHOXNgzRp44IEMJdzZMf5KZFVNdy1rbc1E95caY7Zn\n0b6UUkoppVK0eLFcX7kiJz2ePAk//wxjxsjyd99N/nnWWu6fcT/Dmw+napGq6d5/sWLQpImMtCex\nd6/MI3/33TBjhnvWHZUrZVXLwEnA/6y1a+Lv3w48aa19ONN3duNj0ZpupZRSKg/77DN48knYvl16\ncJcpAzt2uB9PKU345s9vGLVqFDue3IGnh2fmHtSePdLQe
```

Comment:
Should clean up discussion of the second argument too, now that we're using a different order.